controllers/replace_misconfigured_pods_test.go

```diff
@@ -387,6 +387,82 @@ var _ = Describe("replace_misconfigured_pods", func() {
 		})
 	})
 
+	When("PVC name and PVC spec doesn't match", func() {
```

Comment:
Based on you comment that should be changed (https://github.com/FoundationDB/fdb-kubernetes-operator/pull/868#discussion_r678148810)

```suggestion
	When("PVC name doesn't match", func() {
```